### PR TITLE
[APP-4371] Add multi-line functionality to Input component

### PR DESCRIPTION
**Pull request by CalMlynarczyk**
### Description
Design requires a free-form text field for a new page. Since the field would be so similar to the existing `Input` component, I just added the capability to that component using a flag. UXD specified that the field should be resizable vertically.

### Testing Instructions
Go to the _multiline_ example under the `Input` component.

### Screenshots
Screen Shot 2020-01-23 at 11 59 51

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4371)

### CODE REVIEW

**Comment by angelojcruzjr** [comment]
I think the logic for creating this checks out but if we are going to add this we should redo the layout logic to avoid overflow on the Input label (see below):
Screen Shot 2020-01-23 at 11 25 17 AM

**Comment by CalMlynarczyk** [comment]
@angelojcruzjr Hmm, yea. Built this in FF. Didn't realize it would spaz out in Chrome. I'll look into that.

**Comment by CalMlynarczyk** [comment]
@angelojcruzjr Think I resolved the issue.

File: components/atoms/Input/Input.scss [hunk 1]
```diff
@@ -6,19 +6,36 @@ $input-data-value-color: rgba(255, 255, 255, 0);
...
+      border: 1px solid $pink;
+
+      &:focus,
+      &:active {
+        border: 1px solid $pink;
+      }
+    }
+
+    textarea {
+      outline: 1px solid $pink;
```
Comment (JamesIves) [inline components/atoms/Input/Input.scss]: Does this still work correctly when keyboard tabbing with the focus-visible polyfill?

File: yarn.lock [hunk 2]
```diff
@@ -4828,7 +4828,7 @@ debug@3.1.0, debug@=3.1.0:
   dependencies:
     ms "2.0.0"
 
-debug@3.2.6, debug@^3.1.0, debug@^3.2.5, debug@^3.2.6:
+debug@3.2.6, debug@^3.1.0, debug@^3.2.5:
```
Comment (JamesIves) [inline yarn.lock]: Are these changes intentional?

File: components/atoms/Input/Input.mdx [hunk 3]
```diff
@@ -41,6 +41,40 @@ import {Input} from '@unitedincome/components';
 />
 ```
 
+### Multi-line inputs
+
+You can create a multi-line input similar to an HTML text area. Multi-line inputs do not support the `mask` prop.
```
Comment (JamesIves) [inline components/atoms/Input/Input.mdx]: This is nitpicky, but can we place this below the header for `Masks`? It just reads better with prior context on what that prop does.

**Comment by JamesIves** [comment]
JK You fixed the coverage right before I commented

File: yarn.lock (hunk 2 above)
Comment (CalMlynarczyk) [reply yarn.lock]: Yea, I updated them in `develop` as well.

File: components/atoms/Input/Input.scss (hunk 1 above)
Comment (CalMlynarczyk) [reply components/atoms/Input/Input.scss]: As far as I tested it did.
